treeherder/config/settings.py

```diff
@@ -27,11 +27,11 @@
 GRAPHQL = env.bool("GRAPHQL", default=True)
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = env("TREEHERDER_DJANGO_SECRET_KEY")
+SECRET_KEY = env("TREEHERDER_DJANGO_SECRET_KEY", default='SECRET_KEY')
```

Comment:
Change the default to this:
https://github.com/mozilla/treeherder/blob/master/docker-compose.yml#L23